godot-core/src/builtin/meta/class_name.rs

```diff
@@ -19,7 +19,7 @@ pub struct ClassName {
 }
 
 impl ClassName {
-    pub fn new<T: GodotClass>() -> Self {
+    pub fn of_class<T: GodotClass>() -> Self {
```

Comment:
Maybe call it `of`, consistent with [`std::any::TypeId::of`](https://doc.rust-lang.org/std/any/struct.TypeId.html#method.of).

Reply:
Excellent.

I did this rename because I initially read `ClassName::new::<()>()` as "empty class name", because that's usually what `new` without arguments does. Maybe it was a lack of coffee but this is slightly clearer.

Reply:
Fully agree, thanks for bringing this up!